tools/cgrep.py

```diff
@@ -1,93 +0,0 @@
-# Copyright 2011 Google Inc.
```

Comment:
That idea came from vklimovs's email:
"cgrep is fairly small, so let's treat it as first party script (perhaps move out of tools folder)."

That was fine for us, so I tossed it into capirca/. If you'd like it moved back into tools/, I don't think it's a problem either way.